recipes/recipes_emscripten/xz/recipe.yaml

```diff
@@ -19,6 +19,15 @@ requirements:
   - automake
   - libtool
 
+tests:
+- script:
+    - test -f $PREFIX/lib/liblzma.a
```

Comment:
Just FYI if you want to try something new, you can use the `"package_content"` tests.

E.g.

```
- package_content:
    lib: [liblzma.a]
    include: [lzma.h]
```

The benefit is that they would also work on Windows (not really relevant here), and you can easily use globs, too. They run faster as well, if you don't have any other tests.

Reply:
Cool! Let's try it

Reply:
@wolfv could this be a bug??
```
 │ │ ╭─ Package content test
thread 'main' panicked at src/package_test/content_test.rs:151:21:
not implemented: lib_as_globs for target platform: EmscriptenWasm32
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
Ah, it's fixed in a later version of rattler-build :)

Reply:
Good to know. I'll save this for next update then.